content/documentation/staging/faq/general/google-cloud-private-cluster.adoc

```diff
@@ -0,0 +1,15 @@
+---
+title: What do I need to run Kiali in a private cluster?
+weight: 60
+---
+:icons: font
+:sectlinks:
+:linkattrs:
+
+Private clusters have higher network restrictions. Kiali needs your cluster to allow `TCP` traffic between the Kubernates API service and the Istio Control Plane namespace in both `8080` and `15000` ports. Features like `Health` and `Envoy Dump` won't be workinig as expected.
```

Comment:
Kubernates --> Kubernetes
namespace in both --> namespace, for both the
Features like `Health` and `Envoy Dump` won't be workinig as expected -->  This is required for features such as  `Health` and `Envoy Dump` to work as expected.